StandingOvation/StandingOvation.java

```diff
@@ -0,0 +1,107 @@
+import java.io.*;
+import java.util.*;
+import java.io.PrintWriter;
+public class StandingOvation{
+	public static void main(String arg[]){
+		new StandingOvation();
+	}
+	
+	public StandingOvation(){

```

Comment:
The constructor must initialize the object, I don't think it's a good idea to perform the WriteFile operations here